tests/playwright-test/web-server.spec.ts

```diff
@@ -457,6 +457,32 @@ test('should send Accept header', async ({ runInlineTest, server }) => {
   expect(acceptHeader).toBe('*/*');
 });
 
+test('should follow redirects', async ({ runInlineTest, server }) => {
+  server.setRedirect('/redirect', '/redirected-to');
+  server.setRoute('/redirected-to', (req, res) => {
+    res.end('<html><body>hello</body></html>');
+  });
+  const result = await runInlineTest({
+    'test.spec.ts': `
+      import { test, expect } from '@playwright/test';
+      test('connect to the server', async ({baseURL, page}) => {
+        await page.goto('http://localhost:${server.PORT}/redirect');
+        expect(await page.textContent('body')).toBe('hello');
+      });
+    `,
+    'playwright.config.ts': `
+      module.exports = {
+        webServer: {
+          command: 'node ${JSON.stringify(SIMPLE_SERVER_PATH)} ${server.PORT}',
```

Comment:
```suggestion
          command: 'exit 1',
```
I think we can do this to make clear that we should never run the command, since reuseExistingServer is true.

Reply:
This could also be done for some of the other tests, e.g., `should send Accept header` or `should be able to use an existing server when reuseExistingServer:true`. Should I also add it for those tests? If so, should that be done in the same, or in a separate commit?